sdk/test/opentelemetry/sdk/configurator_test.rb

```diff
@@ -87,6 +87,13 @@
           _(configurator_resource_attributes).must_equal(expected_resource_attributes)
         end
       end
+
+      it 'cleans up whitespace in user provided resources' do
+        OpenTelemetry::TestHelpers.with_env('OTEL_RESOURCE_ATTRIBUTES' => 'important_foo=x, important_bar=y') do
```

Comment:
Should this trim values as well? 
```suggestion
        OpenTelemetry::TestHelpers.with_env('OTEL_RESOURCE_ATTRIBUTES' => ' important_foo=x, important_bar=y ') do
```

Reply:
I thought about this but decided against it - I could add it in, not sure what other otel instrumentation agents do.

Reply:
I believe this should be using Baggage format so should be trimming spance then URL decoding https://github.com/open-telemetry/opentelemetry-go/blob/main/sdk/resource/env.go#L92

Reply:
This is my first time ever writing ruby and I haven't been able to run the tests locally (got ruby 2 installed), but I have made the changes and hopefully they make sense.

Reply:
wait sorry I messed up the order, trim first! committing in a moment